src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -270,10 +270,13 @@ def __init__(self, config):
         if self.serverCert and self.serverKey and self.myproxySrv:
             self.proxy = self.setupMyProxy()
 
-        # Build a request string
-        self.reqStr = ('(OpSys == "LINUX" ) && (Arch == "INTEL" || Arch == "X86_64") '
-                       '&& stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) '
-                       '&& ((REQUIRED_OS=="any") || stringListMember(GLIDEIN_REQUIRED_OS, REQUIRED_OS))')
+        # Build a request string.  All CMS resources match DESIRED_Sites on the START
```

Comment:
@bbockelm "All CMS resources match DESIRED_Sites on the START expression side". Where is the START expression evaluated? Where is the second expression evaluated?

If any site is eligible to run jobs, then why not just remove this part of the expression and leave only the REQUIRED_OS + GLIDEIN_REQUIRED_OS validation?